pkg/shell/credentials.js

```diff
@@ -255,12 +256,14 @@ define([
                     dfd.resolve();
                 })
                 .fail(function(ex) {
```

Comment:
```err: "out"``` means that the stderr data is included in output, and thus goes to the stream() function, rather than anywhere else.